src/signal.cr

```diff
@@ -126,4 +126,6 @@ fun __crystal_sigfault_handler(sig : LibC::Int, addr : Void*)
   LibC._exit sig
 end
 
-LibExt.setup_sigfault_handler
+{% unless flag?(:without_sigfault_handler) %}
```

Comment:
Same here: I chilled to build `libcrystal.a` thought it should be relatively easy.